tests/rules/taint_exception.py

```diff
@@ -0,0 +1,242 @@
+# This test aims to exercise various exception paths.
+
+def throw_exits(input):
+  clean = None
+  raise RuntimeError()
+
+  # ok:
+  sink = clean
```

Comment:
`sink = clean` ? Or `sink(clean)` ?

Reply:
I thought I updated all this. Thanks. I'll do another round of query and replace.